src/GlobalStyles.js

```diff
@@ -1,12 +1,23 @@
 import { createGlobalStyle } from 'styled-components/macro'
-
 import { getThemeColors } from './services/getThemeColors'
 
 export default createGlobalStyle`
   * {
     box-sizing: border-box;
   }
 
+  #root {
+    text-align: center;
+    display: grid;
+    grid-template-rows: ${({ userLoggedIn }) => userLoggedIn ? '110px' : '55px'} auto 55px;
+    max-width: 375px;
+    position: fixed;
```

Comment:
I don't think it's necessary to have the whole app `fixed`.
I think you can remove `position`, `left` and `top` here.